### PR TITLE
IBX-2335: Changes after rebranding review

### PR DESCRIPTION
**Pull request by damianz5**
https://issues.ibexa.co/browse/IBX-2335

Changes after rebranding review
tested on php7.4 and php8.1

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -43,19 +43,19 @@
...
     "autoload": {
         "psr-4": {
-            "EzSystems\\EzPlatformAutomatedTranslation\\": "lib",
-            "EzSystems\\EzPlatformAutomatedTranslationBundle\\": "bundle",
             "Ibexa\\AutomatedTranslation\\": "src/lib/",
             "Ibexa\\Bundle\\AutomatedTranslation\\": "src/bundle/",
-            "Ibexa\\Contracts\\AutomatedTranslation\\": "src/contracts/"
+            "Ibexa\\Contracts\\AutomatedTranslation\\": "src/contracts/",
+            "EzSystems\\EzPlatformAutomatedTranslation\\": "lib",
+            "EzSystems\\EzPlatformAutomatedTranslationBundle\\": "bundle"
```
Comment (alongosz) [inline composer.json:50]: `src/lib` and `src/bundle`, right?

File: src/bundle/Form/Data/TranslationAddData.php [hunk 2]
```diff
@@ -15,13 +15,13 @@ class TranslationAddData extends BaseTranslationAddData
     /** @var string|bool */
     protected $translatorAlias;
 
-    /** @return mixed */
+    /** @return string|bool */
```
Comment (alongosz) [inline src/bundle/Form/Data/TranslationAddData.php:18]: why bool?

**Review by alongosz — CHANGES_REQUESTED** [review]

File: src/bundle/Form/Data/TranslationAddData.php (hunk 2 above)
Comment (damianz5) [reply src/bundle/Form/Data/TranslationAddData.php:18]: looks like the form type for translation can be [checkbox](https://github.com/ibexa/automated-translation/blob/main/src/bundle/Form/Extension/TranslationAddType.php#L58) which will result in bool or [choice](https://github.com/ibexa/automated-translation/blob/main/src/bundle/Form/Extension/TranslationAddType.php#L74) which will results in string - depends on how many translation client providers was registered

File: composer.json (hunk 1 above)
Comment (damianz5) [reply composer.json]: Fixed

File: phpstan-baseline.neon [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	ignoreErrors:
+		-
+			message: "#^Access to protected property Ibexa\\\\AdminUi\\\\Form\\\\Data\\\\ContentTranslationData\\:\\:\\$content\\.$#"
+			count: 4
+			path: src/bundle/Form/Extension/ContentEditType.php
+
+		-
+			message: "#^Cannot access property \\$identifier on Ibexa\\\\Contracts\\\\Core\\\\Repository\\\\Values\\\\ContentType\\\\FieldDefinition\\|null\\.$#"
+			count: 2
+			path: src/bundle/Form/Extension/ContentEditType.php
```
Comment (adamwojs) [inline phpstan-baseline.neon:11]: This indicated that we are missing dependencies in composer.json

**Review by adamwojs — CHANGES_REQUESTED** [review]

File: phpstan-baseline.neon (hunk 3 above)
Comment (damianz5) [reply phpstan-baseline.neon]: Fixed

**Review by konradoboza — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Comment by micszo** [comment]
Checkbox hasn't got v4 style and cannot be toggled.

Screenshot 2022-02-24 at 16 03 18

**Comment by micszo** [comment]
The issue has been reported separately: https://issues.ibexa.co/browse/IBX-2433.

**Review by micszo — APPROVED** [review]
QA Approved on Ibexa Commerce v4.0.2 with branch.

**Pull request merged**